template.yml

```diff
@@ -1426,6 +1428,84 @@ Resources:
       Dimensions:
         - Name: LambdaFunctionArn
           Value: !GetAtt ImageTransformLambdaFunction.Arn
+  # Image Manipulation Lambda
+  WavWrapLambdaIamRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      ManagedPolicyArns:
+        # Image operations are always fetching from the artifact bucket
+        - !Ref ArtifactBucketReadOnlyAccessPolicy
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+        - arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess
+      Tags:
+        - Key: Project
+          Value: Porter
+        - Key: prx:cloudformation:stack-name
+          Value: !Ref AWS::StackName
+        - Key: prx:cloudformation:stack-id
+          Value: !Ref AWS::StackId
+  WavWrapLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: lambdas/WavWrapLambdaFunction/
+      Description: >-
+        WAV wrap a source audio file
+      Environment:
+        Variables:
+          S3_DESTINATION_WRITER_ROLE: !GetAtt S3DestinationWriterRole.Arn
+      Handler: index.handler
+      Layers:
+        - !Ref AwsXraySdkLambdaLayerVersionArn
+        - !Ref NpmSharpLambdaLayerVersionArn
+        - !Ref WavefileLambdaLayerVersionArn
+      MemorySize: 3008
+      Role: !GetAtt WavWrapLambdaIamRole.Arn
+      Runtime: nodejs10.x
+      Tags:
+        Project: Porter
+        prx:cloudformation:stack-name: !Ref AWS::StackName
+        prx:cloudformation:stack-id: !Ref AWS::StackId
+      Timeout: 900
```

Comment:
Wasn't sure about good timeout and `MemorySize` for this, so I went max on both.

Reply:
The actual mpeg analysis and wav wrapping are fast operations, and all of it is done in memory.

Downloading the file, and uploading again are the time sinks - so in many ways this should perform more like a copy or analysis task rather than a transcode.